### ISSUE TITLE
fix: 🤔 sd-scrollable screen reader should announce action of arrow click

### ISSUE THREAD

**Issue by smfonseca** [solid-design-system/solid#1683]
## Current behavior
When using screen readers, if the user clicks the arrows, there should be an announcement for something like "scrolling down/up". 
Currently there is only an announcement for "scroll to top/end".

Question: Does it make sense for screenreader users to get informed about the scrolling?

Clarify with a11y expert (Manuel)

## Expected behavior
When clicking the arrows, the performed action is announced ("scrolling up/down").

## Steps to reproduce
- Go to the [sd-scrollable component](https://solid-design-system.fe.union-investment.de/docs/?path=/docs/components-sd-carousel--docs)
- Activate voiceover
- Click the down/ up arrow

## DoR
- [x] Item has business value
- [ ] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by smfonseca** [comment]
Addressed in this [PR](https://github.com/solid-design-system/solid/pull/2109).
Whenever there's an action, "scrolled" is announced.
